packages/playwright/src/reporters/merge.ts

```diff
@@ -288,6 +289,11 @@ function mergeConfigureEvents(configureEvents: JsonEvent[], rootDirOverride: str
   };
 }
 
+function dedupByName<T extends { name: string }>(a: T[], b: T[]) {
```

Comment:
I don't really like deduping by name, because similarly named projects from different bots could be vastly different. I don't see an easy way to achieve what we want right now: from one side, we don't want 10 chromium projects, but from the other side we cannot just dedupe them.

I guess, we should just list tests without filters once in the watch mode and drop this PR.